### ISSUE TITLE
Email forwarding not editable when the user also has Titan email. 

### ISSUE THREAD

**Issue by dolgelukkig** [Automattic/wp-calypso#50181]
After adding Titan email a user is not able to remove/edit an email forward. 

#### Steps to reproduce

- I added an email forward to my domain name
- I added Titan email to my domain name
- The email forward is not showing in my account anymore

#### What I expected

I  expected the forward to be deletable or editable within my account even though Titan was active.  

#### What happened instead

Only Titan settings are visible and editable. 

#### Browser / OS version

MacOS Catalina Version 10.15.7 Chrome browser Version 88.0.4324.150 (Official Build) (x86_64)

#### Screenshot / Video

https://d.pr/i/Zt90vb

**Comment by JoshuaGoode** [comment]
Overall, this is expected behavior as using a full email service such as Titan replaces, and disables, Email Forwarding options.

Email Forwards cannot be used, or routed properly, when an email service is used.  The process of setting a full email service up involves adding the service's MX DNS records and removing the MX record for the forwarding email server. 

When a user purchases G Suite or Titan, this is done automatically for them.

There shouldn't be a need to edit the forwards as they are not in use or active when using a service such as Titan.

To summarize, users cannot use both Email Forwarding and a full email service at the same time and there is no way to.

**Comment by JoshuaGoode** [comment]
Marking this as `[Closed] By Design` for now.  However, feel free to add any questions or concerns related there should be some type of design or interface improvement.

**Comment by arunsathiya** [comment]
@JoshuaGoode That is correct. 👍🏽 However, the context that lead to this GitHub issue is that, an user added an email forwarding to their account and then purchased Titan. They are trying to remove that email forwarding record now, but we don't see an option on Calypso to do that anymore. 

Wendie started a discussion on P2 as well - pcyqBp-1C-p2

**Comment by JoshuaGoode** [comment]
When a user changes their MX records or purchases something like Titan that changes them, forwarding is disabled automatically so there's no need to access or edit the old forwards. They don't do anything at that point.

Essentially, when the `smtp-fwd.wordpress.com` MX record is not present, the forward addresses are disabled and 
`wordpress.com/email/{$domain}/forwarding/{$domain}` is no longer accessible.

I suppose we don't delete them automatically in the event they want to switch back to forwarding but, unless I'm missing something, there's no reason for the user to edit the forwards.
